app/src/main/res/values/appraisals.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="highest_stat_att">attack</string>
+    <string name="highest_stat_def">defence</string>
```

Comment:
You're right, its "defense" in my pogo. Is it really "defence" in yours @MarcLandis ?